plugins/web/active/HTTP_Methods_and_XST@OWTF-CM-008.py

```diff
@@ -11,24 +11,10 @@
 
```

Comment:
Keep the import `from framework.lib.general import get_random_str` with the other imports at the top of the file.